### ISSUE TITLE
Question/Suggestion re handling of heading styles from HTML

### ISSUE THREAD

**Issue by jonathannz** [jgm/pandoc#2173]
Hello,
I've been working (with a coding friend) on a script that will convert, both ways, between MultiMarkdown (MMD) and Rich Text Format (RTF).
For the RTF to MMD conversion... We are using Apple's TextUtil to convert from RTF to HTML, and then Pandoc to get from HTML to MMD. There's an issue, and I don't know whether it's the way we're instructing Pandoc, or a limitation in Pandoc, or something else... so I am posting it here for advice.
The RTF to HTML conversion converts the text sizes in the RTF into styles. Typically, in my case, the smallest font size (happens to be 12) is the body text. Everything else is a heading of varying levels. So the HTML looks like this:
```
<style type="text/css">
    p.p1 {margin: 0.0px 0.0px 16.1px 0.0px; font: 24.0px Times; -webkit-text-stroke: #000000}
    p.p2 {margin: 0.0px 0.0px 14.0px 0.0px; font: 14.0px Times; -webkit-text-stroke: #000000}
    p.p3 {margin: 0.0px 0.0px 12.0px 0.0px; font: 12.0px Times; -webkit-text-stroke: #000000}
    p.p4 {margin: 0.0px 0.0px 14.9px 0.0px; font: 18.0px Times; -webkit-text-stroke: #000000}
    span.s1 {font-kerning: none}
  </style>
```
In the above example, what is represented is this:
```
Style > Size > HTML heading level > MMD heading level
p1 = 24px = h1 = #
p2 = 14pm = h3 = ##
p3 = 12px = body = normal text.
p4 = 16.1px = h3 = ###
```
Pandoc makes nothing of this though.
So what I end up with is an MMD where all the headings, of whatever level, are just a line of bold text.
Is there a way for Pandoc to parse this style data into MMD heading syntax?
If not, is there another way someone can suggest for doing this conversion from RTF to MMD ?
Thank you.

If the answer is NO to all of the above, is this something that could be incorporated into Pandoc? Although, having said that, dealing with headings is so rudimentary I suspect there's already a way to do this, and I've simply over-looked it?

**Comment by mb21** [comment]
Pandoc is about [semantic](http://en.wikipedia.org/wiki/Semantic_HTML) content, not styling, so it ignores almost all CSS. You can get from MMD to RTF directly with Pandoc, but not the other way around since Pandoc doesn't have an RTF reader. Your task is rather difficult because I don't think RTF has a notion of a header (like HTML's `h1` and `h2`), but only of "very very large and bold text" and "large and bold text" etc...

btw, this more of a question than an issue and as such belongs on [pandoc-discuss](https://groups.google.com/forum/#!forum/pandoc-discuss), thx!

**Comment by jonathannz** [comment]
Thanks mb21. I am new to Github. I have no idea how to move this post, nor did I know there was a discuss option. FYI: My posted ended up here because I clicked the [Contact](https://github.com/contact) link on the Pandoc site. Which then said,

> ### Need help with jgm/pandoc?
> You should get in contact with @jgm. You could [open an issue](https://github.com/jgm/pandoc/issues).

I looked on the page located at @jgm but it was not obvious to me how to make contact. I don't suppose there's away to move this post. So I'll consider reposting it where you have suggested. Cheers.
